data/db.json

```diff
@@ -4,7 +4,7 @@
             {
                 "startTime": "2022-08-20 12:00:00",
                 "endTime": "2022-08-20 14:30:00",
-                "body": "{\"foodItems\":\"[{\\\"name\\\":\\\"Arbi Masala Soya Korma Rajma Sambher  Jeera Rice Roti Sirka Onion Dahi\\\",\\\"type\\\":\\\"veg\\\"}]\",\"codeUsed\":false,\"codeExpired\":true,\"multipleMealsAllowed\":false,\"mealAvailedCount\":0}",
+                "body": "{\"foodItems\":\"[{\\\"name\\\":\\\"Arbi Masala Soya Korma Rajma Sambher  Jeera Rice Roti Sirka Onion Dahi\\\",\\\"type\\\":\\\"veg\\\"}]\",\"codeUsed\":false,\"codeExpired\":true,\"multipleMealsAllowed\":false,\"mealAvailedCount\":0}"
```

Comment:
about removing the `,` here it was causing the below error
![image](https://user-images.githubusercontent.com/89068816/198572583-24ea3f6d-20f1-44df-839c-9b9c6d922562.png)

Reply:
its fine, i'll let that pass